File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/DescriptionIDorConceptIDRequest.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for descriptionIDorConceptIDRequest complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="descriptionIDorConceptIDRequest">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="description_id" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *         &lt;element name="concept_id" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *         &lt;element name="stablishment_id" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "descriptionIDorConceptIDRequest", propOrder = {
    "descriptionId",
    "conceptId",
    "stablishmentId"
})
public class DescriptionIDorConceptIDRequest {

    @XmlElement(name = "description_id", required = true)
    protected String descriptionId;
    @XmlElement(name = "concept_id", required = true)
    protected String conceptId;
    @XmlElement(name = "stablishment_id", required = true)
    protected String stablishmentId;

    /**
     * Gets the value of the descriptionId property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getDescriptionId() {
        return descriptionId;
    }

    /**
     * Sets the value of the descriptionId property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setDescriptionId(String value) {
        this.descriptionId = value;
    }

    /**
     * Gets the value of the conceptId property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getConceptId() {
        return conceptId;
    }

    /**
     * Sets the value of the conceptId property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setConceptId(String value) {
        this.conceptId = value;
    }

    /**
     * Gets the value of the stablishmentId property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getStablishmentId() {
        return stablishmentId;
    }

    /**
     * Sets the value of the stablishmentId property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setStablishmentId(String value) {
        this.stablishmentId = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/descriptionIDorConceptIDRequest.php
<?php

namespace Semantikos\ClientBundle\API;


class descriptionIDorConceptIDRequest
{

    /**
     * @var string $description_id
     */
    protected $description_id = null;

    /**
     * @var string $concept_id
     */
    protected $concept_id = null;

    /**
     * @var string $stablishment_id
     */
    protected $stablishment_id = null;

    /**
     * @param string $description_id
     * @param string $concept_id
     * @param string $stablishment_id
     */
    public function __construct()
    {      
    }

    /**
     * @return string
     */
    public function getDescription_id()
    {
      return $this->description_id;
    }

    /**
     * @param string $description_id
     * @return descriptionIDorConceptIDRequest
     */
    public function setDescription_id($description_id)
    {
      $this->description_id = $description_id;
      return $this;
    }

    /**
     * @return string
     */
    public function getConcept_id()
    {
      return $this->concept_id;
    }

    /**
     * @param string $concept_id
     * @return descriptionIDorConceptIDRequest
     */
    public function setConcept_id($concept_id)
    {
      $this->concept_id = $concept_id;
      return $this;
    }

    /**
     * @return string
     */
    public function getStablishment_id()
    {
      return $this->stablishment_id;
    }

    /**
     * @param string $stablishment_id
     * @return descriptionIDorConceptIDRequest
     */
    public function setStablishment_id($stablishment_id)
    {
      $this->stablishment_id = $stablishment_id;
      return $this;
    }

}
